railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -12,10 +12,10 @@
     <%- else -%>
       <%- if gemfile_entries.any? { |m| m.name == 'turbolinks' } -%>
     <%%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
```

Comment:
Did you happen to explore this for stylesheets? Even with async the pages would still appear to load faster?